vllm/attention/backends/hpu_attn.py

```diff
@@ -542,6 +548,13 @@ def forward(
             block_list = attn_metadata.block_list if attn_metadata \
                 and attn_metadata.block_list is not None else None
 
+            common_args = self.common_attention_args(block_list, key_cache,
+                                                     value_cache,
+                                                     attn_metadata.block_size)
```

Comment:
Why move the common_args out?

Reply:
I will move it back. I was going to add additional parameter for left padding argument, but doesn't seem that we need it.